CHANGELOG.rst

```diff
@@ -6,10 +6,16 @@ Changelog
 
     The Dutch version of this changelog can be found :ref:`here <changelog-nl>`.
 
-3.0.8 (2025-05-??)
+3.0.8 (2025-05-23)
 ==================
 
-(unreleased)
+Regular bugfix release.
+
+**New features**
+
+* [:backend:`4933`] Added support for adding Cosign V2 information to registration email templates.
```

Comment:
Can we rephrase this as "Fixed missing Cosign v2 information for registraton email templates." and move it to the bugfixes section?

I want to avoid creating the impression that adding new features to patch releases is okay.